src/util/strings.ts

```diff
@@ -0,0 +1,12 @@
+export const toCamelCaseStatus = (str = '') =>
+  str
+    .toLowerCase()
+    .split(' ')
+    .map((s, index) => {
+      if (index === 0) {
+        return s;
+      }
+
+      return s.charAt(0).toUpperCase() + s.slice(1);
+    })
+    .join('');
```

Comment:
Removed `join` and updated `map` to `reduce`
```suggestion
    .reduce((acc, s, index) => {
      if (index === 0) {
        return s;
      }

      return acc + s.charAt(0).toUpperCase() + s.slice(1);
    })
```

Reply:
@shanpriyan 

Thanks for your suggestion. Just updated in the latest commit.